src/main/kotlin/blackjack/model/Hand.kt

```diff
@@ -1,19 +1,14 @@
 package blackjack.model
 
-class Hand() {
-    private val hold: Cards = Cards(setOf())
+class Hand(val cards: List<Card>) {
+    val size get() = cards.size
 
-    val cards
-        get() = hold.cards
-
-    fun numberOfCards() = hold.cards.size
-
-    fun addCard(card: Card) = this.hold.add(card)
+    operator fun plus(card: Card): Hand = Hand(this.cards + card)
 
     fun calculatePoints(): Int {
-        val countOfAce = hold.cards.filter { it.rank.value == 1 }.size
+        val countOfAce = cards.filter { it.rank.value == 1 }.size
```

Comment:
Instead of comparing values, it might be better to compare the enum itself directly.